framework/include/interfaces/FunctorInterface.h

```diff
@@ -221,3 +230,25 @@ FunctorInterface::defaultFunctor(const std::string & /*name*/)
 {
   return nullptr;
 }
+
+template <typename T>
+void
+FunctorInterface::checkFunctorSupportsSideIntegration(const std::string & name, bool qp_integration)
+{
+  const std::string functor_name = deduceFunctorName(name);
+  const auto & functor = getFunctor<T>(name);
```

Comment:
any way we can retrieve this support without the templating?
probably not but removing templating from the check would be nice